ci/php.yml

```diff
@@ -10,18 +15,37 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    
+    strategy:
+      matrix:
+        php-version: ['7.2', '7.3', '7.4']
 
     steps:
     - uses: actions/checkout@v2
 
+    # Setup PHP environment.
+    # Docs: https://github.com/shivammathur/setup-php/blob/master/README.md    
+    - name: Set up PHP ${{ matrix.php-version }}
+      uses: shivammathur/setup-php@dd27d4eb8158e1dd5bd52a143a7468d7a7fb302a
+      with:
+        php-version: ${{ matrix.php-version }}
+        # extension-csv: mbstring, intl # (optional) specify the extensions you want to enable/install.
+        # ini-values-csv: short_open_tag=On # (optional) specify custom php.ini values.
+        # coverage: xdebug # (optional) specify coverage driver.
+        # pecl: false # (optional) use PECL as fallback to install extensions.
```

Comment:
@MasterOdin 
Please update the optional inputs and the description
```
# extensions: mbstring, intl # (optional) Specify the extensions you want to add or remove.
# ini-values: short_open_tag=On # (optional) Specify the values you want to add to php.ini.
# coverage: xdebug # (optional) Specify the code coverage driver you want to setup.
# tools: pecl # (optional) Specify the tools you want to setup
```

Reply:
done